src/git.md

```diff
@@ -277,6 +287,28 @@ tradeoff. The main advantage is the generally linear commit history. This
 greatly simplifies bisecting and makes the history and commit log much easier
 to follow and understand.
 
+## Keeping things up to date
+
+Using `git pull upstream master` regularly will keep your master branch
+up-to-date, you will also want to rebase your feature branches up-to-date as
```

Comment:
does git pull upstream master only work if you are on the master branch? if so, i can add `git checkout master` in the snippet below to be exceedingly clear

Reply:
Well, it will "work" on other branches, it just doesn't do what you expect (I think it will make a merge commit by default). Yes, it would be great to have `git checkout master`.